config/routes.rb

```diff
@@ -6,6 +6,8 @@
   mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
   mount GovukPublishingComponents::Engine, at: "/component-guide" if defined?(GovukPublishingComponents)
 
+  get "learn-to-drive-a-car", to: 'tasklist#show'
```

Comment:
I believe you missing `/` before the `learn-to-drive-a-car`

Reply:
Good spot, thanks.  It miraculously works either way though :-D